forage-android/src/main/java/com/joinforage/forage/android/ui/DerivedCardInfo.kt

```diff
@@ -0,0 +1,20 @@
+package com.joinforage.forage.android.ui
+
+import com.joinforage.forage.android.model.USState
+
+/**
+ * Represents card information derived from the user's current
+ * Primary Account Number (PAN) input value.
+ */
+interface DerivedCardInfoInterface {
+    /**
+     * The US state that issued the EBT card, derived from the Issuer Identification Number (IIN),
+     * also known as BIN (Bank Identification Number). The IIN is the first 6 digits of the PAN.
+     */
+    var usState: USState?
```

Comment:
Any particular reason we wanted this to be `var` (mutable)? From what I can see, you've been using `DerivedCardInfo.usState` in a strictly read-only / immutable way so far (which is awesome!). Could be a nice little win to make it `val` instead.